### ISSUE TITLE
Background brightness is not being updated

### ISSUE THREAD

**Issue by madonius** [kennylevinsen/wldash#28]
When wldash is opened and the brightness is adjusted with the keyboard keys¹ the bar does not change.
Restarting wldash displays the updated value.

The same issue can not be observed with the volume. The displayed bar is being updated in "real time".

¹ sway config: `bindsym XF86MonBrightnessDown exec light -U 10`

**Comment by kennylevinsen** [comment]
Yeah, brightness is not currently monitored. That will require fsnotify configured on the /sys/ files.

The PulseAudio widgets gets the updates from libpulse.

**Comment by madonius** [comment]
Should I close the issue?

**Comment by kennylevinsen** [comment]
Nah, doesn't hurt to have it open as a feature request.
